monkey/monkey_island/cc/services/ransomware_report.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict, List
+
+from monkey_island.cc.services.reporting.report import ReportService
+
+
+def get_propagation_stats() -> Dict:
+    scanned = ReportService.get_scanned()
+    exploited = ReportService.get_exploited()
+
+    return {
+        "num_scanned_nodes": len(scanned),
+        "num_exploited_nodes": len(exploited),
+        "count_per_exploit": _get_exploit_counts(exploited),
```

Comment:
It seems weird to have two of these named "num" and one named "count". "num" and "count" basically mean the same thing, and I don't think the distinction here is valuable for the consumer of the API.

I think "num_per_exploit", "num_hosts_per_exploit", or "num_nodes_per_exploit" would be better.

I'm also wondering if "hosts" would be better than "nodes", but I don't feel too strongly about this.